samples/user-interface/haptics/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("com.example.platform.sample")
+}
+
+
+android {
+    namespace = "com.example.platform.ui.haptics"
+    buildFeatures {
+        viewBinding = true
```

Comment:
Are you using this? I believe it's all compose now

Reply:
No, removed.